cdc/sink/mq/producer/kafka/kafka.go

```diff
@@ -369,7 +368,6 @@ func NewKafkaSaramaProducer(
 func AdjustConfig(
```

Comment:
It looks can be renamed to `AdjustOptions`.

Reply:
done